packages/editor/templates/Hello World/contracts/HelloWorld.sol

```diff
@@ -1,19 +1,23 @@
-// The source code is for a Solidity version
-// greater than 0.5.10
+// Specifies that the source code is for a version 
+// of Solidity greater than 0.5.10
 pragma solidity ^0.5.10;
 
+// A contract is a collection of functions and data (its state) 
+// that resides at a specific address on the Ethereum blockchain.
 contract HelloWorld {
-    // The keyword "public" makes variables
-    // accessible from other contracts
+    // The keyword "public" makes variables accessible from other contracts
+    // and creates a function that other contracts can call to access the value
     string public message;
```

Comment:
Btw, not only other contracts can call this function, right? Wouldn't be simple to say something on the lines that simply setting a property as public, allows you to use outside the contract itself (through web3js or other contracts per example) 🤔